src/java.compiler/share/classes/javax/annotation/processing/Filer.java

```diff
@@ -66,7 +66,7 @@
  * <p>The file creation methods take a variable number of arguments to
  * allow the <em>originating elements</em> to be provided as hints to
  * the tool infrastructure to better manage dependencies.  The
- * originating elements are the types or packages (representing {@code
+ * originating elements are the classes or interfaces or packages (representing {@code
```

Comment:
would maybe be slightly better as "classes, interfaces or packages"

Reply:
Update in subsequent commit.